site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+  // If the feature is available, take action
+}
+```
+
+To request allowing users to sign in to the IdP from the RP, do the following,
+for example:
+
+```javascript
+const credential = await navigator.credentials.get({
+  identity: {
+    providers: [{
+      configURL: 'https://idp.example/anything.json',
+      clientId: '********',
+      nonce: '******'
+    }]
+  }
+});
+const { token } = credential;
+```
+
+The `providers` property takes an array of [`IdentityProvider`
+objects](https://fedidcg.github.io/FedCM/#dictdef-identityprovider) that have
+the following properties:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>configURL</code> (required)</td>
+    <td>A full path of the IdP config file.</td>
+  </tr>
+  <tr>
+     <td><code>clientId</code> (required)</td>
+     <td>The RP's client identifier, issued by the IdP.</td>
+  </tr>
+  <tr>
+    <td><code>nonce</code> (optional)</td>
+    <td>A random string to ensure the response is issued for this specific request. Prevents replay attacks.</td>
+  </tr>
+</table>
+
+The browser handles sign-up and sign-in use cases differently depending on the
+existence of `approved_clients` in the response from [the accounts list
+endpoint](#accounts-list-endpoint). The browser will only display [the RP's
+privacy policy and terms of service](#client-metadata-endpoint) in the dialog if
+`approved_clients` isn't provided or does not include the RP's `clientId` and
+the user hasn't previously signed up for the RP in this browser.
+
+<figure class="float-right screenshot" style="max-width:300px">
+{% Video
+   src="video/YLflGBAPWecgtKJLqCJHSzHqe2J2/Qx48SEGIEqi5OtPE9ogn.mp4",
+   width="280", autoplay="true"
+%}
+  <figcaption>A user signs into an RP using FedCM</figcaption>
+</figure>
 
-Read detailed instructions on how to implement and [participate in the
-third-party origin trial](/blog/fedcm-origin-trial) for FedCM for support details.
+When the RP calls `navigator.credentials.get()`, the following activities take
+place:
+
+1. The browser sends requests and fetches several documents:
+    1. [The well-known file](#well-known-file) and [an IdP config
+       file](#idp-config-file) which declare endpoints.
+    2. [An accounts list](#accounts-list-endpoint).
+    3. Optional: URLs for the RP's privacy policy and terms of service,
+       retrieved from the [client metadata endpoint](#client-metadata-endpoint).
+2. The browser displays the list of accounts that the user can use to sign-in,
+   as well as the terms of service and privacy policy if available.
+3. Once the user chooses an account to sign in with, a request to [the ID
+   assertion endpoint](#id-assertion-endpoint) is sent to the IdP to retrieve a
+   token.
+4. The RP can validate the token to authenticate the user.
+
+{%
+  Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/8EzrPvIwOsuuzdKJUJ6E.png",
```

Comment:
We have the exact same image above. As a matter of fact, I think there's some duplication to explain the API flow. Do you think we should keep both? Just want to raise the question since the canonical page has been updated multiple times and the duplication could be unintentional.

Reply:
The reason being this is far below enough from the other image and we repeat the info exchange from the RP perspective again here. I thought it's good idea to repeat the image here rather than forcing the reader to go back and forth to look back at the diagram.